parser_library/src/workspaces/workspace_configuration.cpp

```diff
@@ -316,9 +316,13 @@ void workspace_configuration::process_processor_group_and_cleanup_libraries(
 
 bool workspace_configuration::process_program(const config::program_mapping& pgm, std::vector<diagnostic_s>& diags)
 {
-    const auto key = std::make_pair(pgm.pgroup, utils::resource::resource_location());
-    if (!m_proc_grps.contains(key))
-        return false;
+    std::optional<proc_grp_id> grp_id;
+    if (pgm.pgroup != "*NOPROC*")
```

Comment:
make something like `constexpr std::string_view noproc_group_id = "*NOPROC*";` global variable?

Reply:
It's now a `static constexpr` member of `workspace_configuration`